install.sh

```diff
@@ -295,7 +298,7 @@ echo -e "${RESET}"
 echo_info "Updating crew package information...\n"
 # Without setting LD_LIBRARY_PATH, the mandb postinstall fails
 # from not being able to find the gdbm library.
-export LD_LIBRARY_PATH=$(crew const CREW_LIB_PREFIX | sed -e 's:CREW_LIB_PREFIX=::g')
+export LD_LIBRARY_PATH=$(LD_LIBRARY_PATH=/usr/local/lib${LIB_SUFFIX}:/usr/lib${LIB_SUFFIX}:/lib${LIB_SUFFIX} crew const CREW_LIB_PREFIX | sed -e 's:CREW_LIB_PREFIX=::g')
```

Comment:
Can you give me a convincing reason why we need this line?

Reply:
On i686, at least in the container, install.sh starts with LD_LIBRARY_PATH to /usr/lib:/lib.
Invoking crew in this line invokes ruby. Ruby depends upon the newer libgcc which is in /usr/local/lib. 
However, owing to the LD_LIBRARY_PATH, if first looks in /usr/lib, where it finds an older libgcc. 
This causes crew to fail running `crew const CEW_LIB_PREFIX`, which in turn causes the install script to fail.
Merely having the LD_IBRARY_PATH fixed for this one line avoids this problem.

Reply:
On newer systems, the system libgcc doesn't cause an issue here.

Reply:
It will still find the newer libgcc first since `/usr/local/lib` is the higher priority path.  I guess I still don't understand why we need the extra `:/usr/lib${LIB_SUFFIX}:/lib${LIB_SUFFIX}` added to the path.

Reply:
I thought it would too... but it doesn't. My build logs show a libgcc error.

Reply:
Why do we need anything except `export LD_LIBRARY_PATH=$(LD_LIBRARY_PATH=/usr/local/lib${LIB_SUFFIX})`?

Reply:
> Why do we need anything except `export LD_LIBRARY_PATH=$(LD_LIBRARY_PATH=/usr/local/lib${LIB_SUFFIX})`?

We may be calling system libraries at this point since core isn't installed yet, and those libraries are not in /usr/local...

Reply:
Packages already know how to find system libraries, otherwise they wouldn't work at all.

Reply:
It's a question of library priority. We do NOT want the system libraries preferred to chromebrew priorities, since the system libraries for libgcc are much much older on i686.

Reply:
> It will still find the newer libgcc first since `/usr/local/lib` is the higher priority path. I guess I still don't understand why we need the extra `:/usr/lib${LIB_SUFFIX}:/lib${LIB_SUFFIX}` added to the path.

@uberhacker This is wrong. The `/etc/ld.so.conf` shows what the default library search path priorities are. 
on armv7l:
```
chronos@rpi4b-armv7l /usr/local/lib/crew/packages $ cat /etc/ld.so.conf 
# ld.so.conf autogenerated by env-update; make all changes to
# contents of /etc/env.d directory
/lib
/usr/lib
/usr/local/lib
```
on i686:
```
chronos@cheekon-i686 /usr/local/lib/crew/packages $ cat /etc/ld.so.conf 
# ld.so.conf autogenerated by env-update; make all changes to
# contents of /etc/env.d directory
/lib
/usr/lib
/usr/local/lib
```
on x86_64:
```
chronos@cheekon-x86_64 /usr/local/lib/crew/packages $ cat /etc/ld.so.conf 
# ld.so.conf autogenerated by env-update; make all changes to
# contents of /etc/env.d directory
/lib64
/usr/lib64
/usr/local/lib64
/lib
/usr/lib
/usr/local/lib
```
On ALL of these images, the system libs are higher in the search hierarchy. I'm just trying to override this once when first running crew to get the crew library folder.

Reply:
This is the error I see: 

![image](https://user-images.githubusercontent.com/1096701/193279257-9697f1fd-f1a1-4954-81cc-295d643bedde.png)

Reply:
(And again, this issue is ONLY for i686.)

Reply:
This is now simplified.

Reply:
Nice!